frontend/micro-ui/web/micro-ui-internals/packages/modules/payment/src/configs/UICustomizations.js

```
@@ -85,70 +85,60 @@
+        return (
+          <span className="link">
             <Link
               to={`/${window.contextPath}/citizen/payment/open-view?tenantId=${row.tenantId}&businessService=${row.businessService}&consumerCode=${row.consumerCode}`}
             >
               {String(value ? (column.translate ? t(column.prefix ? `${column.prefix}${value}` : value) : value) : t("ES_COMMON_NA"))}
             </Link>
-          </span> 
+          </span>
+        );
       }
     },
```

Biome

[error] This code is unreachable

... because either this statement, ...

... this statement, ...

... this statement, ...

... this statement, ...

... this statement, ...

... this statement, ...

... this statement, ...

... this statement, ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)